### PR TITLE
Runner: Init - Add superbench runner class

### PR DESCRIPTION
**Pull request by abuccts**
__Major Revisions__
* init runner class

### CODE REVIEW

File: tests/runner/test_runner.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    def tearDown(self):
+        """Hook method for deconstructing the test fixture after testing it."""
+        shutil.rmtree(self.output_dir)
+
+    def test_set_logger(self):
+        """Test log file exists."""
+        expected_log_file = Path(self.runner._output_dir) / 'sb-run.log'
+        self.assertTrue(expected_log_file.is_file())
+
+    def test_run(self):
```
Comment (cp5555) [inline tests/runner/test_runner.py:39]: It's a little bit strange to check NotImplementedError

**Review by cp5555 — APPROVED** [review]

File: tests/runner/test_runner.py (hunk 1 above)
Comment (abuccts) [reply tests/runner/test_runner.py]: removed

File: tests/runner/test_runner.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+from omegaconf import OmegaConf
+
+from superbench.runner import SuperBenchRunner
+
+
+class RunnerTestCase(unittest.TestCase):
+    """A class for runner test cases.
+
+    Args:
+        unittest.TestCase (unittest.TestCase): TestCase class.
```
Comment (guoshzhao) [inline tests/runner/test_runner.py:20]: This is inheriting not args, right?

**Review by guoshzhao — APPROVED** [review]

File: tests/runner/test_runner.py (hunk 2 above)
Comment (abuccts) [reply tests/runner/test_runner.py]: updated

**Pull request merged**